src/main/java/com/purbon/kafka/topology/model/users/platform/KsqlServerInstance.java

```diff
@@ -23,6 +23,10 @@ public String commandTopic() {
     return String.format("_confluent-ksql-%s_command_topic", ksqlDbId);
   }
 
+  public String internalTopics() {
```

Comment:
I like the idea to generalise all internal topics and not only the command topic. Do you mind adapting the command topic reference as well? If not used anymore, should not be here.